modules/local/clean_work_dirs.nf

```diff
@@ -0,0 +1,20 @@
+// Import generic module functions
+include { saveFiles } from './functions'
+
+params.options = [:]
+
+/**
+ * Clean intermediate directories.
+ */
+process clean_work_dirs {
+    tag { sample_id }
+    label "local"
+
+    input:
+    tuple val(sample_id), val(directory)
+
+    script:
+    """
+    clean_work_dirs.sh "${directory}"
```

Comment:
Since each process is in its own file now, would it make sense to move the bin scripts back into the processes? I think that would be better at this point.

Reply:
Yes, I agree.  The modules should be self-contained and not rely on scripts in the bin directory.  I'll try and make this change.

Reply:
I just pushed a commit that removed the .sh files and merged in that code to the module files.  I tested all three aligners and they worked.